docs/kubefirst/installer/config-file.md

```diff
@@ -38,19 +38,3 @@ config:
 ```bash 
 kubefirst init  -c ./values.yaml
 ```
-
-## Sample of Local Mode - Github
```

Comment:
Why did we remove this?

Reply:
Because we don't support config files on local

Reply:
we actually do, it's in the local viper docs https://github.com/kubefirst/kubefirst/blob/main/configs/viperConfig.go#L33 , the filename:

https://github.com/kubefirst/kubefirst/blob/main/configs/viperConfig.go#L40

Reply:
That was my comprehension, that we did, just not all the same configuration options.

Reply:
out of clarification, we were under refactoring, and decided to reduce the refactor scope, having cloud and local following the same viper config is in the radar for a future refactor window.